index.html

```diff
@@ -9,6 +9,12 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello world</h1> 
+  <h1 class="title">Front-End Tools</h1> 
+  <div class="tech-card">
+    <img class="tech-card-image" src="assets/react_icon.svg"></img>
+    <h2 class="tech-card-title">React-Js</h2>
+    <p class="tech-card-release">Release: 2013</p>
```

Comment:
you could use a span here since it's a small piece of text and not an actual paragraph